app/components/pages-menu/style.scss

```diff
@@ -6,3 +6,23 @@
     border-radius: 2px;
     text-decoration: none;
 }
+
+.pages-nav {
+    &.left {
+       text-align: left;
+       float:left
```

Comment:
In testing, text-align did not move the navigation, floating was the only way I could get it to move